runtime/onert/backend/gpu_cl/Backend.h

```diff
@@ -60,11 +60,9 @@ class Backend : public ::onert::backend::Backend
     auto tr = std::make_shared<ClTensorRegistry<TensorManager>>(tm);
 
     InferenceContext::CreateInferenceInfo create_info;
-    create_info.precision = environment->IsSupported(CalculationsPrecision::F16)
-                              ? CalculationsPrecision::F16
-                              : CalculationsPrecision::F32;
-    create_info.storage_type = GetFastestStorageType(environment->device().GetInfo());
-
+    create_info.precision = CalculationsPrecision::F32;
+    create_info.storage_type =
+      GetStorageTypeWithMinimalMemoryConsumption(environment->device().GetInfo());
```

Comment:
This change doesn't seem to be for this PR.

Reply:
AFAIk, test does not run test.

Reply:
@YiHyunJin Could you please rewrite commit and PR comment to represent the modification  ?